themes/theme-gmd/components/ProductCard/index.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { bin2hex } from '@shopgate/pwa-common/helpers/data';
 import Link from '@shopgate/pwa-common/components/Link';
 import Ellipsis from '@shopgate/pwa-common/components/Ellipsis';
-import { ITEM_PATH, MapPriceHint } from '@shopgate/engage/product';
+import { ITEM_PATH, MapPriceHint, OrderQuantityHint } from '@shopgate/engage/product';
```

Comment:
It seems, that only the `ProductCard` from the theme components was extended with `OrderQuantityHint` and `MapPriceHint`, but not the one which we expose to extensions through the `ThemeContext`. Was this intentional?

We use this component within the `LiveshoppingWidget` and the `ProductSlider` component which is exposed through the `ThemeContext`.